.mailmap

```diff
@@ -0,0 +1,7 @@
+Aleksa Sarai <asarai@suse.de> <cyphar@cyphar.com>
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sur@coreos.com> <sergiusz.urbaniak@gmail.com>
+Stephen Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
+xiekeyang <xiekeyang@huawei.com> <keyang.xie@gmail.com>
```

Comment:
I think you [still][1] want an additional line for `xiekeyang@users.noreply.github.com`.  As it stands you get:

```
$ git describe 
v0.2.0-43-g380a18c
$ git shortlog -se | grep xiekeyang
    21  xiekeyang <xiekeyang@huawei.com>
    22  xiekeyang <xiekeyang@users.noreply.github.com>
```

With an additional line you can collapse those together:

```
$ ENTRIES=$(echo 'xiekeyang <xiekeyang@huawei.com> <xiekeyang@users.noreply.github.com>'; cat .mailmap)
$ echo "${ENTRIES}" | sort --ignore-case | uniq >.mailmap
$ git shortlog -se | grep xiekeyang
    43  xiekeyang <xiekeyang@huawei.com>
```

[1]: https://github.com/opencontainers/image-tools/pull/186#pullrequestreview-65098077

Reply:
Done.